docs/tutorial/install.rst

```diff
@@ -38,3 +38,15 @@ Install Microceph
     for i in $(seq 1 3); do
         lxc exec microceph-$i -- sh -c 'snap install microceph'
     done
+
+Update management
+~~~~~~~~~~~~~~~~~
+
+Updates are not managed from within microceph itself. As such, users are the ones who will manage them. Thus, automatic snap updates should be postponed indefinitely.
+This can be achieved with the following command (Supported with snapd from version 2.58 onwards):
+
+.. code-block:: shell
+
+    for i in $(seq 1 3); do
+        lxc exec microceph-$i -- sh -c 'snap refresh --hold microceph'
```

Comment:
This command is specific to running micro ceph in a lxd container. You should simply state that the snap refresh --hold command is the correct command. The example can reference to running within a lxd container, but a baremetal machine wouldn't use the lxc part of the command.